main.go

```diff
@@ -36,6 +43,13 @@ func main() {
 	if err := flag.CommandLine.MarkHidden("omit-timestamps"); err != nil {
 		panic(err)
 	}
+
+	flag.Usage = func() {
+		PrintUsage()
+		fmt.Fprintln(os.Stderr, "Flags:")
```

Comment:
The purpose of this is the same as the above comment -- the desire is to have a standalone, shared [usage message](https://en.wikipedia.org/wiki/Usage_message) and the external caller does not have access to `flag`.

Reply:
I think it's fine if the external caller were to document these flags as well

Reply:
I'm not entirely sure I'm following what is meant by "the external caller does not have access to flag". I'm mostly just looking at the code and seeing that it seems like it can be moved so that the usage printing isn't split across two places. And since it's not used in any other package, it doesn't need to be exported.

I don't care that much and don't wanna bike shed a minor CL that is improving the not great existing usage text.

Reply:
The request to share the logic has been dropped thus this is no longer applicable.

context: go/drcao